src/main/java/com/solutions/shop/controller/UserController.java

```diff
@@ -13,17 +13,16 @@ public class UserController {
     @Autowired
     private UserService userService;
 
-    @PostMapping("user/{login},{password}")
+    @PostMapping("user/{login}/{password}")
```

Comment:
Here are no such path variables like {login} {password}. Remove them from mapping.

@PostMapping("user/{login}/{password}") -> @PostMapping("user")

Reply:
Should be also @RequestBody @Valid UserDto user